docs/google/best_practices.md

```diff
@@ -263,12 +263,62 @@ commuting operators.
 See the function `cirq.stratified_circuit` for an automated way to organize gates
 into moments with similar gates.
 
+### Qubit picking
+
+On current NISQ devices, qubits cannot be considered identical.  Different
+qubits can have vastly different performance and can vary greatly from day
+to day.  It is important for experiments to have a dynamic method to
+pick well-performing qubits that maximize fidelity of the experiment.
+There are several techniques that can assist with this.
+
+*   Analyze calibration metrics:  performance of readout, single-qubit, and
+two-qubit gates are measured as a side effect of running the device's
+calibration procedure.  These metrics can be used as a baseline to evaluate
+circuit performance or identify outliers to avoid.  This data can be inspected
+programmatically by retrieving metrics from the [API](calibration.md) or
+visually by applying a `cirq.Heatmap` to that data or by using the built-in
```

Comment:
link to the visualizaing metrics tutorial: https://quantumai.google/cirq/tutorials/google/visualizing_calibration_metrics

Reply:
Done.